custom_components/mammotion/services.yaml

```
@@ -75,31 +74,25 @@ start_mow:
       default: 2
       selector:
         select:
+          translation_key: "ultra_waves"
```

```suggestion
          translation_key: "ultra_wave"
```
This would match the app